src/Debug.elm

```diff
@@ -56,12 +56,18 @@ crash = Native.Debug.crash
 
 {-| Watch a particular value in the debugger. Say we want to know the value of
 a variable called `velocity` because it may not be updated correctly. Adding
-`Debug.watch` allows us to name the value and show it with the debugger.
+`Debug.watch` allows us to name the value and show it with the debugger. The
+result of evaluating such an expression is exactly the same as if not having
+the call to `Debug.watch` at all.
 
 	  Debug.watch "velocity" velocity == velocity
 
-Notice that the result of evaluating this expression is exactly the same as
-not having the expression at all. That means it's easy to add to any value.
+That means it's easy to add `Debug.watch` to any value.
+
+Note that calling `Debug.watch` on a signal is not useful. Instead, it needs
+to be mapped into the signal (to act on the contained value). So if you want
+to watch a timer signal, instead of `Debug.watch "time" <| Time.every second`
```

Comment:
Both `every` and `second` are in the `Time` module so it doesn't make much sense to have one qualified and one not. For the sake of explanation I would drop the `Time.` part.